lib/rules/value-keyword-case/index.js

```diff
@@ -65,14 +65,16 @@ const rule = function (expectation, options) {
         }
 
         // Ignore keywords within `url` and `var` function
-        if (node.type === "function" && (valueLowerCase === "url" || valueLowerCase === "var" || valueLowerCase === "counter" || valueLowerCase === "counters" || valueLowerCase === "attr")) {
-          return false
+        if (node.type === "func" && (valueLowerCase === "url" || valueLowerCase === "var" || valueLowerCase === "counter" || valueLowerCase === "counters" || valueLowerCase === "attr")) {
+          // Prevent further descending, but don't return false as that stops all walking
+          node.walk = null
```

Comment:
`node.walk` is a PostCSS API. I don't think it's a good idea, to break PostCSS API.